### PR TITLE
feat: Individual imports

### PR DESCRIPTION
**Pull request by imprashast**
Adding support to be able to import individual components to make the client bundle tree shake properly.

### CODE REVIEW

File: build.mjs [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import esbuild from "esbuild";
+import { glob } from "glob";
+import fs from 'fs';
+import ts from 'typescript';
+
+const components = glob.sync("packages/**/src/index.ts");
+const indexPath = "packages/index.ts";
```
Comment (BalbinaK) [inline build.mjs:7]: Should we expose dead-toggle, clickable, affix and so on from _helpers/ folder as separate components, too?

Comment (imprashast) [reply build.mjs:7]: Good point, didn't know that 👍

Comment (imprashast) [reply build.mjs]: These helper components needs to be moved out of _helpers and should follow the same pattern (packages/**/src) for typescript to work properly. I'll create a new task for that and do those in the next PR.

**Review by BalbinaK — APPROVED** [review]
LGTM

**Pull request merged**